src/App.jsx

```diff
@@ -25,6 +26,7 @@ function App() {
         <Route path="/login" component={""} />
         <Route path="/games/memorygame" component={MemoryGame} />
         <Route exact path="/games/puzzle" component={PuzzlePage} />
+        <Route path="/games/quizgame" component={QuizGamePage} />
```

Comment:
@tmansuroglu can you please rename your component to MemoryGamePage. you can see that there is an inconsistency here in the naming

Reply:
sure